hedera-node/docs/design/services/services.md

```diff
@@ -1,35 +1,267 @@
+  messages in real-time.
+
+### Use Cases
+
+- **Supply Chain Tracking**
+- **Decentralized Applications (dApps)**
+- **Audit Logs**
+
+**Full Documentation:
+** [Consensus Service](consensus-service/consensus-service.md)
+
+## Hedera Token Service (HTS)
+
+### Overview
+
+The Hedera Token Service (HTS) enables the creation, management, and transfer of
+native tokens on the Hedera network. It supports a wide variety of use cases,
+including fungible and non-fungible tokens (NFTs).
+
+### Key Functions
+
+- **Token Creation:** Supports the creation of tokens with customizable
+  properties such as supply, decimal places, and freeze, wipe, and KYC status.
+- **Transfers and Ownership:** Securely manage token transfers and ownership,
+  including bulk transfers.
+- **Minting and Burning:** Efficiently mint and burn tokens as needed.
+
+### Use Cases
+
+- **Asset Tokenization**
+- **Loyalty Programs**
+- **NFTs and Digital Collectibles**
+
+**Full Documentation:** [Token Service](token-service)
+
+## Hedera Smart Contract Service
+
+### Overview
+
+The Hedera Smart Contract Service allows developers to deploy and interact with
+smart contracts on the Hedera network, enabling decentralized applications (
+dApps) with complex logic and automated processes.
+
+### Key Functions
+
+- **Contract Deployment:** Deploy smart contracts written in Solidity.
+- **Contract Execution:** Execute contract functions with predictable and secure
+  results.
+- **Interoperability:** Integrate with other services like HTS and HCS for
+  enhanced functionality.
+
+### Use Cases
+
+- **Decentralized Finance (DeFi)**
+- **Automated Workflows**
+- **Tokenized Assets**
+
+**Full Documentation:** [Smart Contract Service](smart-contract-service)
+
+## Hedera File Service
+
+### Overview
+
+The Hedera File Service (HFS) provides a decentralized file storage solution,
+allowing users to create, update, and delete files on the Hedera network. It's
+often used in conjunction with other services to store data such as contract
+bytecode or token metadata.
+
+### Key Functions
+
+- **File Management:** Create, update, and delete files on the network.
+- **Storage of Smart Contracts:** Store smart contract bytecode and other
+  associated data.
+- **Data Availability:** Ensure data is available and tamper-proof across the
+  network.
+
+### Use Cases
+
+- **Document Management**
+- **Data Storage for dApps**
+- **Smart Contract Storage**
+
+**Full Documentation:** [File Service](file-service/file-service.md)
+
+## Hedera Network Service
+
+### Overview
+
+The Hedera Network Service encompasses the foundational components that ensure
+the security, scalability, and reliability of the Hedera network. It includes
+the mechanisms for managing accounts, processing transactions, and ensuring
+network integrity.
+
+### Key Functions
+
+- **Account Management:** Manage user accounts, including creation, updates, and
```

Comment:
Network Admin service allows querying of accounts, but not create / update / delete

Reply:
Thanks updated 👍